### PR TITLE
Fix coals build queue

### PR DESCRIPTION
**Pull request by nkprince007**

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -1,39 +1,14 @@
...
-coala==0.9.1
-coala-utils==0.5.1
-codecov==2.0.5
-colorama==0.3.7
-colorlog==2.10.0
+coala-bears==0.10
 coverage==4.3.4
 defusedxml==0.5.0
 Django==1.8
-django-ckeditor==5.1.1
```
Comment (ghoshbishakh) [inline dev-requirements.txt]: @nkprince007 sure we dont need these?

Comment (nkprince007) [inline dev-requirements.txt]: Oops.. 
Forgot that..

Comment (nkprince007) [inline dev-requirements.txt]: Guess, it's there.. but in a different line.

Comment (ghoshbishakh) [inline dev-requirements.txt]: So the rest you removed are just fine right? I am not rechecking

**Comment by nkprince007** [comment]
Yes, I'm pretty sure. FWIW, merge it.

**Comment by ghoshbishakh** [comment]
@nkprince007 Thanks a lot man!! 😄

**Pull request merged**